server/test/article.spec.js

```diff
@@ -365,8 +368,48 @@ describe('Article email share', () => {
       .post(`/api/v1/article/${validSlug}/share`)
       .set({ Authorization: `Bearer ${userToken}` })
       .send(mockDataShare.validShareObject);
-
     expect(res).to.have.status(200);
     expect(res.body).to.have.property('message').eql('Article shared successfully');
   });
 });
+
+describe('Delete Article', () => {
+  before(async () => {
+    const userResponse = await chai
+      .request(app)
+      .post('/api/v1/auth/login')
+      .send({
+        email: 's.david@example.com',
+        password: 'soldier'
+      });
+
+    authorToken = userResponse.body.data.token;
+  });
+
+  const validSlug = 'the-fattest-girl-3654677788';
+  const fakeSlug = 'David is so annoying it-12345678';
+
+  it('should check if the user is the author of the article', async () => {
+    const res = await chai.request(app)
+      .delete(`/api/v1/article/${validSlug}`)
+      .set({ Authorization: `Bearer ${userToken}` });
+    expect(res).to.have.status(401);
+    expect(res.body.message).to.eql('Action restricted to author of article');
```

Comment:
Hi, @Chiazokam should an admin be given the privilege to delete an article?

Reply:
No. Only the creator of an article can delete it. Admins should be able to suspend articles but that is another feature

Reply:
@Chiazokam Alright